Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -545,9 +551,12 @@ def main(cls, args, repository, **kwargs):
         if not branch_point:
             return 1
 
-        result = cls.create_commit(args, repository, **kwargs)
-        if result:
-            return result
+        if args.commit is None:
+            args.commit = repository.config().get('webkitscmpy.auto-create-commit') == 'true'
```

Comment:
My only concern here is that by default, this would be `false` (which changes the current behavior)

The way we've dealt with this for the other options reference in `pull_request.py` is have `Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py` set a default in `PROJECT_CONFIG_OPTIONS` (which applies to all repositories) and then have `metadata/git_config_extension` override this default for a specific project. I think in this case, the `PROJECT_CONFIG_OPTIONS` being `true` would be sufficient, we could consider prompting the user in `git-webkit setup` to override this (like we do with rebasing), but I'm not sure that's necessary.